packages/state-manager/src/sagas/messages/messages.transform.ts

```diff
@@ -1,38 +1,66 @@
-import { EntityState } from '@reduxjs/toolkit'
 import { createTransform } from 'redux-persist'
+import { channelMessagesAdapter } from '../publicChannels/publicChannels.adapter'
+import { ChannelMessage } from '../publicChannels/publicChannels.types'
 import { StoreKeys } from '../store.keys'
-import { messageSendingStatusAdapter, messageVerificationStatusAdapter } from './messages.adapter.ts'
+import { messagesBaseAdapter, messageSendingStatusAdapter, messageVerificationStatusAdapter, publicChannelsMessagesBaseAdapter } from './messages.adapter.ts'
 import { MessagesState } from './messages.slice'
-import { MessageVerificationStatus } from './messages.types'
+import { MessageVerificationStatus, PublicChannelsMessagesBase } from './messages.types'
 
 export const MessagesTransform = createTransform(
   (inboundState: MessagesState, _key) => {
     return { ...inboundState }
   },
   (outboundState: MessagesState, _key) => {
     const messageVerificationStatus = Object.values(outboundState.messageVerificationStatus.entities)
-
     const updatedMessageVerificationStatus: MessageVerificationStatus[] = messageVerificationStatus.reduce((result, status: any) => {
-      const entry: MessageVerificationStatus = {
-        ...status,
-        isVerified: status.isVerified ? status.isVerified : status.verified
-      }
-      if (entry['verified']) delete entry['verified']
+      if(status.isVerified !== undefined ||
+      status.verified !== undefined
+      ){
+        const entry: MessageVerificationStatus = {
+          ...status,
+          isVerified: status.isVerified ? status.isVerified : status.verified
```

Comment:
So in some cases we will have both verified and isVerified right?

Reply:
yes

Reply:
that's super confusing

Reply:
which one is true?

Reply:
isVerified

Reply:
so why we keep verified?

Reply:
we don't keep verified, when someone will update from old version with verified keys it will transform to isVerified here